core/src/test/java/io/confluent/rest/SaslTest.java

```diff
@@ -227,9 +227,8 @@ private void assertMetricsCollected() {
             "Request latency metrics should be measurable",
             metricValue instanceof Double);
         double latencyMaxValue = (double) metricValue;
-        assertTrue(
-            "Metrics should be collected (max latency shouldn't be 0)",
-            latencyMaxValue != 0.0);
+        assertNotEquals("Metrics should be collected (max latency shouldn't be 0)",
+            0.0, metricValue);
```

Comment:
nit: Thanks to @rigelbm I'm a newly-minted adherent to [the Rectange Rule](https://github.com/google/google-java-format/wiki/The-Rectangle-Rule) - I think this will read slightly better as
```
assertNotEquals(
    "Metrics should be collected (max latency shouldn't be 0)",
    0.0,
    metricValue);
```

Reply:
Oh my, I really dislike Google formatting. Oh well, I'll make the change here even though I think it takes more space and is not more readable.